cpp/tests/io/json_test.cpp

```diff
@@ -2434,6 +2434,55 @@ TEST_F(JsonReaderTest, MapTypes)
           {type_id::LIST, type_id::STRING, type_id::STRING});
 }
 
+TEST_F(JsonReaderTest, JsonLinesRandomDelimiter)
+{
+  using SymbolT = char;
+
+  std::array<SymbolT, 4> delimiter_chars{{'\n', '\b', '\v', '\f'}};
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_int_distribution<> distrib(0, delimiter_chars.size() - 1);
+  SymbolT const random_delimiter = delimiter_chars[distrib(gen)];
+
+  // Test input
+  std::string input             = R"({"col1":100, "col2":1.1, "col3":"aaa"})";
```

Comment:
Are there any tricky edge cases this string should be testing?  Should the string itself be randomized somehow?

Reply:
Good question - this test checks if the JSON tokenization FST is able to handle non-newline delimiter separating rows in JSON lines. The JSON structure per line does not matter for this test.